src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -1,2 +1,26 @@
-package com.kodilla.ecommercee.domain;public class Product {
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "PRODUCT")
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
```

Comment:
To jak z tym Getterami i Setterami robimy?

Reply:
Zostawcie je, bądą potrzebne przy mapperach